hack/gcp-secret-manager/util.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+import click
+from google.auth import default
+from google.auth.exceptions import DefaultCredentialsError
+
+# Test Platform's project in GCP Secret Manager
+PROJECT_ID = "openshift-ci-secrets"
+
+# The YAML config which defines which groups have access to which secret collections.
+CONFIG_PATH = "https://raw.githubusercontent.com/openshift/release/master/core-services/sync-rover-groups/_config.yaml"
```

Comment:
If a user is running this script, it means he has already checked out the `openshift/release` repository and, in turn, this implies that the `core-services/sync-rover-groups/_config.yaml` file is already available on the local filesystem. What about reading it from the filesystem instead of issuing a request to GitHub?

Reply:
Hm, good point...

Reply:
The counter argument is that the user needs to have an up-to-date release repo locally

Reply:
We can keep this as it is, just something to keep in mind.